pipelines/rj_pic__disparos_email/flow.py

```
@@ -0,0 +1,253 @@
+
+        # Cria instância do EmailSender e envia e-mail
+        email_sender = EmailSender()
+        success = email_sender.send_email(
+            to_email=email,
+            subject=email_subject,
+            html_body=html_body,
+            recipient_name=nome
+        )
+
+        if success:
+            print(f"  ✅ Enviado com sucesso")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)